src/google/protobuf/port_def.inc

```diff
@@ -971,8 +971,9 @@
 // globally even though it's only used for PROTOBUF_FIELD_OFFSET.
 #pragma GCC diagnostic ignored "-Winvalid-offsetof"
 #ifdef __clang__
-// TODO(b/255348263): Upgrade our kokoro Ubuntu to use clang 6, then change th 5 to a 6
-static_assert(PROTOBUF_CLANG_MIN(5, 0), "Protobuf only supports Clang 6.0 and newer.");
+static_assert(PROTOBUF_CLANG_MIN(6, 0), "Protobuf only supports Clang 6.0 and newer.");
+#elif defined(__GNUC__)
+static_assert(PROTOBUF_GNUC_MIN(6, 0), "Protobuf only supports GCC 6.0 and newer.");
```

Comment:
7.3.1 is our latest gcc

Reply:
Yep, unfortunately some of our tests are still using 6.0 :(

Reply:
I'll upgrade those tests too to get us onto 7.3